index.html

```diff
@@ -527,6 +527,13 @@ <h3>Proofs</h3>
 in Section <a href="#proof-chains"></a>.
           </dd>
 
+          <dt>nonce</dt>
+          <dd>
+An OPTIONAL string value supplied by the proof creator. An example use of this 
+field is to increase privacy by decreasing linkability to deterministically 
+generated signatures.
```

Comment:
I think this is more accurate:

```suggestion
field is to increase privacy by decreasing linkability that is the result
of deterministically generated signatures.
```

The only linkability decreased is whatever is generated directly from a deterministic signature itself.

Reply:
Thanks for the better wording.